packages/core/src/connectivity/scp-cf/destination/destination-selection-strategies.ts

```diff
@@ -1,9 +1,9 @@
 import { createLogger } from '@sap-cloud-sdk/util';
-import {
+import type {
   AllDestinations,
   DestinationsByType
 } from './destination-accessor-types';
-import { Destination } from './destination-service-types';
+import type { Destination } from './destination-service-types';
```

Comment:
👍 
Do you see some warnings from the IDE or some command?

Reply:
yes my IDE makes a red underline under the type because it seems not to understand it. But it compiles fine.

Reply:
Then another question, why you know you can optimise it by only importing the type, if your IDE cannot understand and suggest the improvement.

Reply:
I did not buy the latest webstorm version so perhaps this is the reason. I agree that you could have a lint rule that checks if you use a import just as a `type` and adjust then the import.